src/app/screens/Publishers/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+import Container from "../../components/Container";
+import PublishersTable from "../../components/PublishersTable";
+
+import { Allowance } from "../../../types";
+import utils from "../../../common/lib/utils";
+import websites from "./websites.json";
+
+function Publishers() {
+  const [data, setData] = useState<Allowance[]>([]);
+  const navigate = useNavigate();
+
+  function navigateToPublisher(id: string) {
+    navigate(`/publishers/${id}`);
+  }
+
+  async function fetchData() {
+    try {
+      const response = await utils.call<{
+        allowances: Allowance[];
+      }>("listAllowances");
+      const allowances = response.allowances.map((allowance) => {
+        if (allowance.enabled && allowance.remainingBudget > 0) {
+          return {
+            ...allowance,
+            badge: {
+              label: "ACTIVE",
+              color: "green-bitcoin",
+              textColor: "white",
+            },
+          };
+        }
+        return allowance;
+      });
+      setData(allowances);
+    } catch (e) {
+      console.error(e);
+    }
+  }
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  return (
+    <Container>
+      <h2 className="mt-12 mb-2 text-2xl font-bold dark:text-white">
+        Your ⚡️ Websites
+      </h2>
+      <p className="mb-6 text-gray-500">
+        Websites where you have used Alby before
+      </p>
+      {data.length > 0 ? (
+        <PublishersTable
+          publishers={data}
+          navigateToPublisher={navigateToPublisher}
+        />
+      ) : (
+        <p>No websites yet.</p>
+      )}
+      <h2 className="mt-12 mb-2 text-2xl font-bold dark:text-white">
+        Other ⚡️ Websites
+      </h2>
+      <p className="mb-6 text-gray-500">Websites where you can use Alby</p>
+      <div className="mb-12">
+        {websites.map(({ title, items }) => (
+          <div className="mb-6" key={title}>
+            <h4 className="mb-4 text-xl font-bold">{title}</h4>
+            <div className="grid gap-4 md:grid-cols-4">
+              {items.map(({ title, subtitle, logo, url }) => (
+                <a key={url} href={url} target="_blank" rel="noreferrer">
+                  <div className="bg-white shadow-md flex space-x-3 h-44 px-4 rounded-lg pt-7 hover:bg-gray-50 cursor-pointer w-full">
```

Comment:
is `h-32` or something maybe enough? I feel like the boxes are pretty big. 
Every description seems to fit except the lightning store one - but there we can shorten the description.